manifests/server.pp

```diff
@@ -0,0 +1,17 @@
+# ex: syntax=puppet si ts=4 sw=4 et
+
+define bind::server (
+    $bogus     = false,
+    $edns      = true,
+    $key       = undef,
+    $transfers = undef,
+    $order     = '10',
```

Comment:
I suspect that the order of server clauses in the bind configuration doesn't matter. If that's the case, I think I would prefer to see the `concat::fragment`'s order hard-coded to a value rather than have it exposed as a parameter. Is the order meaningful in any useful case?